### PR TITLE
add model comparator space to spaces

### PR DESCRIPTION
**Pull request by FarukOzderim**

### CODE REVIEW

File: docs/hub/spaces.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ pinned: false
 
 For additional settings, refer to the [Reference](#reference) section.
 
+### Compare or Clone Models and Spaces
+You can build a space comparing Hugging Face Models and Spaces or create clones of them with [Model Comparator Space Builder](https://huggingface.co/spaces/farukozderim/Model-Comparator-Space-Builder) 🤗.
```
Comment (osanseviero) [inline docs/hub/spaces.md:91]: Maybe you can add a bit more info about this Space and split the information

Something like this maybe?
> 
> ### Comparing different models
> 
> There is a Space that allows you to automatically create a Space to compare different models and Spaces. ...
> 
> ### Duplicating a space
> 
> ...

**Review by osanseviero — COMMENTED** [review]
This is awesome! :rocket: thanks for working on it :muscle:

File: docs/hub/spaces.md (hunk 1 above)
Comment (FarukOzderim) [reply docs/hub/spaces.md:91]: And I will create 2 different headings like you mentioned.

Comment (osanseviero) [reply docs/hub/spaces.md:91]: Sounds good!

Comment (FarukOzderim) [reply docs/hub/spaces.md]: updated the space and description, there is also a new heading that came into my mind :D

**Review by osanseviero — APPROVED** [review]
Very nice, thanks for this!

**Pull request merged**